hummingbird/ml/_container.py

```diff
@@ -136,28 +135,23 @@ def transform(self, *inputs):
         return self._run(self._transform, *inputs, reshape=True)
 
 
-class PyTorchSklearnContainerRegression(PyTorchTorchscriptSklearnContainer):
+class SklearnContainerRegression(SklearnContainer):
     """
     Container mirroring Sklearn regressors API.
     """
 
     def __init__(
         self, model, n_threads, batch_size, is_regression=True, is_anomaly_detection=False, extra_config={}, **kwargs
     ):
-        super(PyTorchSklearnContainerRegression, self).__init__(model, n_threads, batch_size, extra_config)
+        super(SklearnContainerRegression, self).__init__(model, n_threads, batch_size, extra_config)
 
         assert not (is_regression and is_anomaly_detection)
 
         self._is_regression = is_regression
         self._is_anomaly_detection = is_anomaly_detection
 
-    def _predict(self, *inputs):
-        if self._is_regression:
-            return self.model.forward(*inputs).cpu().numpy().ravel()
-        elif self._is_anomaly_detection:
-            return self.model.forward(*inputs)[0].cpu().numpy().ravel()
-        else:
-            return self.model.forward(*inputs)[0].cpu().numpy().ravel()
+    def _predict(self, *input):
```

Comment:
Can you please add  comments on the "pass" ones explaining the broader usage?

Reply:
Done.